interp/src/test.rs

```diff
@@ -1,7 +1,477 @@
 #[cfg(test)]
```

Comment:
As a note clippy doesn't run by default on the tests, you'll need to run `cargo clippy --all --all-targets`